src/components/core/data-list.tsx

```diff
@@ -18,9 +18,9 @@ export function entryFromArray(label: string, array: string[]) {
 export function DataList({ title, data }: DataListProps) {
   return (
     <div className="ctw-space-y-4 ctw-rounded-lg ctw-bg-bg-lighter ctw-p-4">
-      <span className="ctw-text-sm ctw-uppercase ctw-text-content-light">
+      <h4 className="ctw-m-0 ctw-text-sm ctw-uppercase ctw-text-content-light">
```

Comment:
Hmmm why `h4` here? Are there `h3`, `h2` and `h1` above it? I think it is risky to depend on `h4` styling as that is often changed globally in CSS. Instead switch back to `span` and explicitly set the text styles.

Reply:
These changes are for semantics rather than style. I did have to add styles _(like `ctw-m-0`)_ in order to make the heading tags look as the span did.

The other option would be to use aria attributes I suppose
```suggestion
      <span
        className="ctw-text-sm ctw-uppercase ctw-text-content-light"
        role="heading"
        aria-level="4"
      >
```

Reply:
@WesleyKapow for reference this figma screenshot is where the levels came from

<img width="451" alt="Screen Shot 2022-11-15 at 10 12 25 AM" src="https://user-images.githubusercontent.com/6380075/201954928-cd8e2e6e-c9a4-4c07-a8b0-f2eb1ca5ff16.png">

https://www.figma.com/file/BX8WkUMyecKimerE4VjdkB/Medication-Overview?node-id=682%3A6088

Reply:
Ah ok awesome! We are using them correctly then! Carry on!